doc/install_doc_component.html

```diff
@@ -46,10 +46,10 @@
       <p><strong>We recommend that you install a complete scientific Python
        distribution like 64 bit <a class="reference external" href="https://
        store.continuum.io/cshop/anaconda/" target="_blank">Anaconda</a>
-       </strong>. Since it meets all the requirements of nilearn, it will save 
+       </strong>. Since it meets all the requirements of nilearn, it will save
        you time and trouble. You could also check <a class="reference external"
-       href="http://code.google.com/p/pythonxy/" target="_blank">PythonXY</a>
        as an alternative.</p>
+       href="http://code.google.com/p/pythonxy/" target="_blank">PythonXY</a>
```

Comment:
For the record the link is not broken per se, it's just a "This project has moved" page with the up-to-date link to github.io.